src/content/concepts/module-federation.mdx

```diff
@@ -22,7 +23,7 @@ related:
 
 ## Motivation
 
-Multiple separate builds should form a single application. These separate builds should not have dependencies between each other, so they can be developed and deployed individually.
+Multiple separate builds should form a single application. These separate builds act like containers and can expose and consume code between builds, creating a single, unified application.
```

Comment:
`should not have dependencies between each other, so they can be developed and deployed individually` Is a little confusing and makes me thing that dependencies cannot be shared. Not sure what this was originally intended to mean